util/cron/test-cray-xc-gpu-arkouda.bash

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Run arkouda testing on a cray-cs with HDR IB
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-native-gpu.bash
+export CHPL_COMM=none
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="cray-xc-gpu-arkouda"
+
+# setup arkouda
+export CHPL_TEST_ARKOUDA_PERF=false
+source $CWD/common-arkouda.bash
+export ARKOUDA_NUMLOCALES=16
```

Comment:
Don't need/want numlocales for comm=none

Reply:
You're right, I copied that over naively from another script; I've removed it.